mongo_bridge/Data[] data